lib/helpers.js

```diff
@@ -0,0 +1,686 @@
+    *   @param {string} text
+    *   @example {{uppercase text}}
+    *   @returns The string converted
+    */
+module.exports.uppercase = value => value.toString().toUpperCase();
+
+/**
+    *   Replace the coincidence passed in the text with a value using regex
+    *	@param {string} The text to analyze
+    *	@param {string} The string to search in the text
+    *   @param {string} The flag for the regex
+    *	@param {string} The string to replace in the text
+    *   @example {{uppercase text}}
+    *   @returns The string converted
+    */
+module.exports.regexReplace = (text, search, flags, replace) => {
+
+	if(!search)
+		return '';
+
+	const regex = new RegExp(search.toString(), flags.toString());
+
+	return text.replace(regex, replace);
+};
+
+/**
+    *  Strips a host from a URL
+    *  @param   {string} url - The URL which host will be removed.
+    *  @returns The string striped
+    *  @example {{stripHost url}} => /search
+    */
+module.exports.stripHost = URL => URL.toString().replace(/https?:\/\/.*?\//i, '/');
+
+/**
+    *  Camelize a string
+    *  @param   {string} text - The string to camelize
+    *  @returns The string formatted
+    *  @example {{camelize text}}
+    */
+module.exports.camelize = value => utils.sanitizeString(value.toString(), ' ').replace(/(?:^\w|[A-Z]|\b\w)/g, (letter, index) => {
+	return index === 0 ? letter.toLowerCase() : letter.toUpperCase();
+})
+	.replace(/\s+/g, '');
+
+/**
+    *  Sanitize a string
+    *  @param   {string} text - The string to sanitize
+    *  @returns The string formatted
+    *  @example
+    */
+module.exports.sanitize = (value, replace) => utils.sanitizeString(value, replace);
+
+/**
+    *  Modulus block
+    *  Checks if the module of a value is equal to result
+    *  @param {int} index The value to make the operation
+    *  @param {int} mod The modulus value
+    *  @param {int} result The modulus result
+    *  @returns If the condition is true return the content inside of block, otherwise return the content inside of block else
+    *  @example {{#modulus 2 2 0}}True{{/modulus}}
+    */
+module.exports.modulus = function(index, mod, result, block) {
+	if(Number(index) % mod === Number(result))
+		return block.fn(this);
+	return block.inverse(this);
+};
+
+/**
+    *  Convert to html as text preformatted from an object
+    *  @param {object} args- The object to convert
+    *  @returns The html
+    *  @example {{debugString object}}
+    */
+module.exports.debugString = object => new Handlebars.SafeString(`<pre>${JSON.stringify(object, ' ', 4)}</pre>`);
+
+/**
+    *  Convert to html as text preformatted from objects or values
+    *  @param {...mixed} args- The objects or values to convert
+    *  @returns The html
+    *  @example {{debug object1 value1}}
+    */
+module.exports.debug = (...args) => {
+
+	const params = args.slice();
+
+	params.pop();
+
+	let output = '<style>body { position: static!important;}</style>';
+
+	for(const param of params)
```

Comment:
```suggestion
	let output = '<style>body { position: static!important;}</style>';

	for(const param of args)
```

Reply:
Como el ultimo parametro siempre es **_block_** tuve que hacer:
`	const params = args.slice(0, -1);`
Para luego: 
`	for(const param of params)`